### ISSUE TITLE
Support @ngx-translate/core@"^14.0.0" ?

### ISSUE THREAD

**Issue by YipingRuan** [lephyrus/ngx-translate-messageformat-compiler#79]
````
npm ERR! Could not resolve dependency:
npm ERR! peer @ngx-translate/core@"^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0.0" from ngx-translate-messageformat-compiler@4.11.0
````

**Comment by viceice** [comment]
![image](https://user-images.githubusercontent.com/1798109/145957777-89d9ac5d-bd90-4920-be99-a997c97e02e7.png)

**Comment by lephyrus** [comment]
Addressed by #78.
